docs/services/postgresql/add-standby-node.md

```diff
@@ -0,0 +1,39 @@
+# Adding a postgresql hot standby node
```

Comment:
@rohit-dimagi 

Thanks for adding this. 

It looks like this article is orphaned though. You can link this page in https://github.com/dimagi/commcare-cloud/blob/master/docs/index.md to add it to docs (in a new PR)

Reply:
I think this should go under the `postgresql` service docs. So you can add a link in: https://github.com/dimagi/commcare-cloud/blob/master/docs/services/postgresql.md

Reply:
(that's what I meant, I could have probably mean more specific)

Reply:
Noted. This will be updated in upcoming MR.